rules-reviewed/eap7/eap6/tests/data/data-hsearch/HsearchUtil.java

```diff
@@ -127,7 +131,12 @@ public void main(String[] args) {
         BuildContext context = (BuildContext) searchIntegrator;
         context.getIndexingStrategy();
         
-        System.out.println(IndexWriterSetting.TERM_INDEX_INTERVAL + " test");
```

Comment:
OK, that's somewhat understandable, because for this to work, Windup would have to have a fully annotated AST, for which it would have to have a complete compilation classpath. I'm not sure how much name and type resolution Windup currently does.

Anyway, I think this doesn't affect us much.